presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+only consults the underlying file system for files that must be read.
+
+Status
+------
+
+The Iceberg connector is not yet supported for production use.
+
+Compatibility With Spark Iceberg
+--------------------------------
+
+Tests have been written to demonstrate that data written by Presto can be read
+using Spark SQL with the Iceberg format, and vice versa.  See
+`TestSparkCompatibility <https://github.com/prestosql/presto/blob/master/presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestSparkCompatibility.java>`_.
+
+Configuration
+-------------
+
+Iceberg supports the same metastore configuration properties as the Hive connector.
+At a minimum, ``hive.metastore.uri`` must be configured:
+
+.. code-block:: none
+
+    connector.name=iceberg
+    hive.metastore.uri=thrift://localhost:9083
+
+Partitioned Tables
+------------------
+
+Iceberg supports partitioning by specifying transforms over the table columns.
+A partition is created for each unique tuple value produced by the transforms.
+Identity transforms are simply the column name. Other transforms are:
+
+===================================== ================================================================================================
+Transform                             Description
+===================================== ================================================================================================
+``year(ts)``                          The partition value is the integer difference in years between ``ts`` and January 1st. 1970.
+
+``month(ts)``                         The partition value is the integer difference in months between ``ts`` and January 1st. 1970.
+
+``day(ts)``                           The partition value is the integer difference in days between ``ts`` and January 1st. 1970.
+
+``hour(ts)``                          The partition value is a TIMESTAMP rounded down to the nearest hour.
+
+``bucket(x, nbuckets)``               The partition value is an integer hash of ``x``, with a value between 0 and ``nbuckets - 1`` inclusive.
+
+``truncate(s, nchars)``               The partition value is the first ``nchars`` characters of VARCHAR ``s``.
+===================================== ================================================================================================
+
+In this example, the table is partitioned by month and by a hash of the account number::
+
+    CREATE TABLE iceberg.testdb.sample_partitioned (
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR)
+    WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)'])
+
+Deletion by Partition
+---------------------
+
+For partitioned tables, the Iceberg connector supports the deletion of entire
+partitions if the WHERE clause specifies an identity transform of a partition
+column.  Given this table definition::
+
+    CREATE TABLE test_partition_deletion (
```

Comment:
It's better to use a more realistic example, similar to the one above.

Reply:
Changed to be an example using the table above.